cachito/web/models.py

```diff
@@ -150,7 +150,20 @@ def bundle_temp_files(self):
         cachito_bundles_dir = flask.current_app.config['CACHITO_BUNDLES_DIR']
         return os.path.join(cachito_bundles_dir, 'temp', str(self.id))
 
-    def to_json(self):
+    @property
+    def number_of_deps(self):
+        """
+        Get the total number of dependencies for a request.
+
+        :return: count of the number of dependencies
+        :rtype: int
+        """
+        req_dep_cols = request_dependency_table.c
+        num_deps = db.session.query(sqlalchemy.func.count(req_dep_cols.dependency_id)).filter(
```

Comment:
A variable isn't necessary. You can just return it directly.

Reply:
It was looking ugly that way so I decided to put it in a variable. But I can do that too.